code/model/recipients/UserDefinedForm_EmailRecipient.php

```diff
@@ -113,25 +112,29 @@ protected function getRulesConfig() {
 	 */
 	public function getCMSFields() {
 		Requirements::javascript(USERFORMS_DIR . '/javascript/Recipient.js');
-
+        $fields = FieldList::create(Tabset::create('Root')->addExtraClass('EmailRecipientForm'));
+        $multiOptionFields = new DataList('EditableMultipleOptionField');
+		$validEmailFromFields = new DataList('EditableEmailField');
+		$validSubjectFields = new DataList('EditableTextField');
+		
 		// Determine optional field values
-		$form = $this->getFormParent();
-
-		// predefined choices are also candidates
-		$multiOptionFields = EditableMultipleOptionField::get()->filter('ParentID', $form->ID);
-
-		// if they have email fields then we could send from it
-		$validEmailFromFields = EditableEmailField::get()->filter('ParentID', $form->ID);
-
-		// For the subject, only one-line entry boxes make sense
-		$validSubjectFields = ArrayList::create(
-			EditableTextField::get()
-				->filter('ParentID', $form->ID)
-				->exclude('Rows:GreaterThan', 1)
-				->toArray()
-		);
-		$validSubjectFields->merge($multiOptionFields);
-
+		if( $form = $this->getFormParent() ){
+    
+    		// predefined choices are also candidates
+    		$multiOptionFields = EditableMultipleOptionField::get()->filter('ParentID', $form->ID);
+    
+    		// if they have email fields then we could send from it
+    		$validEmailFromFields = EditableEmailField::get()->filter('ParentID', $form->ID);
+    
+    		// For the subject, only one-line entry boxes make sense
+    		$validSubjectFields = ArrayList::create(
```

Comment:
Indentation is a bit weird here. this is probably my fault, but do you mind converting this file to use spaces only for indentation? (aka .editorconfig rules).